Doc/library/turtle.rst

```diff
@@ -1925,6 +1994,24 @@ Window control
 Animation control
 -----------------
 
+.. function:: no_animation()
+
+   Temporarilly disable turtle animation. The code written inside the
+   ``no_animation`` block will not be animated, and once the code block is
+   exited, the drawing will appear.
+
+   .. doctest::
+      :skipif: _tkinter is None
+
+      >>> with screen.no_animation():
+      ...     dist = 2
+      ...     for i in range(200):
+      ...         fd(dist)
+      ...         rt(90)
+      ...         dist += 2
+   .. versionadded:: next
```

Comment:
Ah.. I am so sorry. There should be a newline just above. I messed up my suggestion (see how you wrote the others `versionadded`).

Reply:
This should be fixed now 🙂

Reply:
Thank you :)